playwright/_impl/_page.py

```diff
@@ -817,6 +818,10 @@ async def wait_for_function(
     def workers(self) -> List["Worker"]:
         return self._workers.copy()
 
+    @property
+    def request(self) -> "APIRequestContext":
```

Comment:
Seems unrelated.

Reply:
Since we roll in this PR, this got caught up by it.